src/components/resume/resume.svelte

```diff
@@ -14,7 +14,7 @@
 
   let scaleControl = 1;
 
-  function scaleResume() {
+  function scaleResume(scaleReset: boolean | undefined = undefined) {
```

Comment:
I could just verify whether or not the window height is 400 px like I did in the fitResumeContainer function and do the correct behavior in there without passing an override.